.storybook/common.ts

```diff
@@ -186,3 +187,7 @@ export interface StorybookCustomEvent {
   name: string;
   payload: any;
 }
+
+export function addStorybookListeners(events) {
+  events.forEach(e => addEventListener(e, logEvent));
```

Comment:
I'm assuming that this is attaching to the global window by default? Thoughts on being more explicit about this?